memos/docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      # <app-id>_<web-container-name>_1
+      # e.g. 'btc-rpc-explorer_web_1'
+      # Note that the '_1' at the end is needed
+      APP_HOST: memos_main_1
+      APP_PORT: 5230
+      PROXY_AUTH_WHITELIST: "/api/*"
+
+  main:
+    image: neosmemo/memos:0.21
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}:/var/opt/memos
```

It's kinda confusing, but the ${APP_DATA_DIR} variable is `<path-to-umbrel-install>/app-data/<app-id>`, so to bind mount this to `<path-to-umbrel-install>/app-data/<app-id>/data` we need to change to:

```suggestion
    volumes:
      - ${APP_DATA_DIR}/data:/var/opt/memos
```